src/types/index.ts

```diff
@@ -6,4 +8,111 @@ export interface DashboardLinkConfig {
 
 export interface DashboardConfig extends DashboardLinkConfig {
   slot: string;
-}
\ No newline at end of file
+}
+
+export interface InternationalPatientSummary {
+  [x: string]: any;
```

Comment:
Do we still need this in the type?

Reply:
@ibacher  Thanks for the reviews, I  have addressed the comments. However, for this very one, I have just updated this to entry, when I remove it I will have no property entry on the response;